### ISSUE TITLE
Can't connect a third party Switch Pro controller

### ISSUE THREAD

**Issue by Sistaaa** [GaryOderNichts/Bloopair#80]
Hi, I'm new to hacking a Wii U. I've installed Tiramisu and Bloopair, but I can't connect my third party Switch Pro controller (model MG26Z). Is there a way to fix this?

Thank you in advance.

**Comment by GaryOderNichts** [comment]
Can you connect the controller to a PC or phone and check with which name it shows up?

**Comment by Sistaaa** [comment]
There are a couple of ways to connect the controller.

1. With bluetooth, the name is "GamePadPlus V3"
2. With the receiver, the name is "Microsoft X-Box 360"
3. With the cable, the name is "Microsoft X-Box 360"

**Comment by GaryOderNichts** [comment]
Please test if the following build works:
[30_bloopair.rpx.zip](https://github.com/GaryOderNichts/Bloopair/files/10300154/30_bloopair.rpx.zip)

**Comment by Sistaaa** [comment]
Unfortunately, that didn't work. I guess that build was the same as I had.

**Comment by GaryOderNichts** [comment]
That build had the `GamePadPlus V3` name added, so that Bloopair would identify the controller as a Switch controller.
Does the controller work on a Switch? Is there some special toggle which puts the controller into "Switch mode"?

**Comment by Sistaaa** [comment]
Ah I see. I don't have a Switch, I use the controller with emulators, but it should work with Switch.

I found this online: https://ae01.alicdn.com/kf/H4b5cc5fea7414a068262f5ebe7f386599.jpg
With Android I use A+Home to enter in pairing mode, but for Switch it should be Y+Home. I just tried it with both builds, but without luck.

I searched further and found https://ae01.alicdn.com/kf/S62905ac28f984268bb70e6a2995beeaeq/Originele-Draadloze-Controller-Bluetooth-Voor-Nintendo-Schakelaar-Pro-Pc-PS3-Joystick-Gamepad-Nfc-Professionele-Vertragingsvrije-Gamepad.jpg
It looks like the model I have (MG26Z) only works with the receiver on Switch. So I plugged the receiver on my Wii U and it seems te connect (the lights om my controller stops blinking), but I can't use the controller.

Would it work if you add the "Microsoft X-Box 360" name to the build?
